src/test/java/org/jitsi/meet/test/web/WebParticipantFactory.java

```diff
@@ -178,11 +192,25 @@ else if (participantType == ParticipantType.edge)
             final ChromeOptions ops = new ChromeOptions();
             ops.addArguments("use-fake-ui-for-media-stream");
             ops.addArguments("use-fake-device-for-media-stream");
-            ops.addArguments("disable-extensions");
             ops.addArguments("disable-plugins");
             ops.addArguments("mute-audio");
             ops.addArguments("disable-infobars");
 
+            if(options.getChromeExtensionId() != null)
+            {
+                try
+                {
+                    ops.addExtensions(
+                        downloadExtension(options.getChromeExtensionId()));
+                }
+                catch (IOException e)
+                {
+                    e.printStackTrace();
```

Comment:
what ? shouldn't that fail the test ?

Reply:
It will eventually fail the test probably anyway. How do you want to change it?

Reply:
to wrap in RuntimeException at least